secret-service/pkg/backend/secretbackend_k8s.go

```diff
@@ -220,6 +235,16 @@ func remove(s []string, r string) []string {
 	return s
 }
 
+func insert(s []string, i string) []string {
```

Comment:
More out of curiosity: Would this not be a nice fit to use pointers or do the operation in place instead of using a return?

Reply:
I wanted to stay in line with similar functions of the golang core library like `append()`, which also returns the value of the resulting array, instead of modifying the pointer